src/lib/tooltip/tooltip.ts

```diff
@@ -149,8 +170,12 @@ export class MdTooltip implements OnDestroy {
   /** The message to be displayed in the tooltip */
   @Input('mdTooltip') get message() { return this._message; }
   set message(value: string) {
-    this._message = value;
-    this._setTooltipMessage(this._message);
+    if (this._message) { this._unregisterA11yMessage(this._message); }
+
+    // If the message is not a string (e.g. number), convert it to a string and trim it.
+    this._message = value ? `${value}`.trim() : '';
```

Comment:
This is done to address an issue where numbers are being used as Input and are throwing an error because they do not have `trim()`. This converts the input to string and trims it.